modules/EPMC.py

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/env python
+
+import argparse
+import sys
+import logging
+
+import pyspark
+from pyspark.sql.types import StringType
+from pyspark.sql import SparkSession 
+import pyspark.sql.functions as pf
+from pyspark import SparkContext
+from pyspark.conf import SparkConf
+
+
+# The following target labels are excluded as they were grounded to too many target Ids
+EXCLUDED_TARGET_TERMS = ['TEC', 'TECS', 'Tec', 'tec', '\'', '(', ')', '-', '-S', 'S', 'S-', 'SS', 'SSS',
+         'Ss', 'Ss-', 's', 's-', 'ss', 'U3', 'U6', 'u6', 'SNORA70', 'U2', 'U8']
```

Comment:
Is this list ever likely to change? Also, is this something we receive from Miguel or could it be detected on the fly when ingesting the dataset?

Reply:
I consider it as an interim solution. We are having some plans to improve grounding, once those improvements are in place, having this list and the filter in place we won't need that. 

This list was created based on my analysis showing these labels were notorious for being mapped to too many target IDs. But again, this issue should be treated systematically at the level of grounding. David/Miguel are aware of the issue, but due to time constraints, we could not address the issue.